### PR TITLE
add missing desc stanzas to casks

### PR DESCRIPTION
**Pull request by gdams**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Also had to bump a few versions with broken URLs

### CODE REVIEW

File: Casks/launchy.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
```
Comment (SMillerDev) [inline Casks/launchy.rb:4]: Livecheck can check for `href="downloads/mac/Launchy2.5.dmg"` in https://www.launchy.net/download.php#osx

File: Casks/baiducloud.rb [hunk 2]
```diff
@@ -1,15 +1,16 @@
 cask "baiducloud" do
-  version "2.4.6"
-  sha256 "9ae594d4d6a4581d971ed9fcde7cda62022b04247476a8982802324fa16bee2e"
+  version "3.9.0"
+  sha256 "e23abb9ff8183029adada8384e11946da521a4e3cfba410052b4456d8294c670"
```
Comment (SMillerDev) [inline Casks/baiducloud.rb:4]: Can check for `target-href="https://issuecdn.baidupcs.com/issue/netdisk/MACguanjia/BaiduNetdisk_mac_3.9.0.dmg"` in the homepage

Comment (gdams) [reply Casks/baiducloud.rb:4]: I've been trying with this block but it doesn't seem to work:

```rb
  livecheck do
    url "https://pan.baidu.com/download"
    strategy :page_match
    regex(%r{href=.*?/BaiduNetdisk_mac_(\d+(?:\.\d+)*)\.dmg}i)
  end
```

Comment (gdams) [reply Casks/baiducloud.rb:4]: I'm having major issues with the `baiducloud.rb` app, the download server is very unstable, the website is very hard to scrape. Not sure what the best approach is there

Comment (miccal) [reply Casks/baiducloud.rb:4]: This already exists as a Cask:

https://github.com/Homebrew/homebrew-cask/blob/master/Casks/baidunetdisk.rb

Let's just remove this one:
```
==> Analytics
install: 14 (30 days), 47 (90 days), 259 (365 days)
```

**Comment by miccal** [comment]
`superbeam` is a 32-bit app, so let's remove it:
```
==> Analytics
install: 0 (30 days), 1 (90 days), 7 (365 days)
```

**Comment by miccal** [comment]
Thank you @gdams.

**Pull request merged**